### PR TITLE
CSIP31 ready for review

### PR DESCRIPTION
**Pull request by andersbonielsen**
Test Case AND corpus ready for review (moving fast, combining both reviews and a pull request)

### CODE REVIEW

File: corpora/csip/metadata/amdsec/CSIP31/test-case.xml [hunk 1]
```diff
@@ -11,14 +11,85 @@ XML lists -->
...
+      <!-- Text description of the validation rule -->
+      <description>There should be an XML element named amdSec in the mets.xml</description>
+      <!-- Details of the validation error generated by the message, can be one of: [ ERROR | WARNING | INFO ]-->
+      <error level="WARNING">
+        <!-- Expected error message when the rule is violated. -->
+        <message>The element mets/amdSec does not exist in METS.xml. It SHOULD be there.</message>
+      </error>
+      <corpusPackages>
+        <!-- Details of corpus packages designed to test this validation rule. -->
+        <package name="IP_18000_CSIP31_1" isValid="FALSE">
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/test-case.xml]: I am not sure what the purpose of isValid is? Should it be false if the package should produce a warning or an error? and true if it should produce no warning or error?

File: corpora/csip/metadata/amdsec/CSIP31/test-case.xml [hunk 2]
```diff
@@ -11,14 +11,85 @@ XML lists -->
...
+          <path>/corpus/structure/dmdSec/CSIP31/IP_18000_CSIP31_1</path>
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/test-case.xml]: This path is wrong - should be put in corpora and amdSec, not dmdSec

File: corpora/csip/metadata/amdsec/CSIP31/test-case.xml [hunk 3]
```diff
@@ -11,14 +11,85 @@ XML lists -->
...
+          <!-- Full description of corpus package features. -->
+          <description>A package based on the minimal IP with schemas. Contains no dmdSec</description>
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/test-case.xml]: change dmdSec to amdSec

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
I have made some comments on the commit.
In general there are several places where it says: dmdSec where it should say amdSec.

File: corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Minimal IP with schemas and amdSec-->
+<!-- In this example IDs are carring information to ease understanding - they look like paths to ease understanding, but are just IDs, similar to the naming of namespaces - these IDs can be replaced with information less UUID -->
+<!-- CSIPSTR15 goes like this: We recommend including all schema documents for any structured metadata within package. These schema documents SHOULD be placed in a sub-folder called schemas within the Information Package root folder and/or the representation folder. -->
+<mets xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.loc.gov/METS/"
+  xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:csip="https://DILCIS.eu/XML/METS/CSIPExtensionMETS"
+  xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance schemas/XMLSchema.xsd http://www.loc.gov/METS/ schemas/mets.xsd 
+    http://www.w3.org/1999/xlink schemas/xlink.xsd 
+    https://DILCIS.eu/XML/METS/CSIPExtensionMETS schemas/CSIPExtensionMETS.xsd"
+  OBJID="IP_18000_CSIP17_2" TYPE="Database" csip:CONTENTINFORMATIONTYPE="SIARD2"
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml]: Wrong value in OBJID

File: corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        <FLocat LOCTYPE="URL" xlink:href="schemas/CSIPExtensionMETS.xsd" xlink:type="simple"/>
+      </file>
+    </fileGrp>
+  </fileSec>
+  <structMap TYPE="PHYSICAL" LABEL="CSIP StructMap" ID="StructmapID">
+    <!-- CSIP80 mets/structMap -->
+    <!-- CSIP81 mets/structMap/@TYPE The type attribute of the structural map (structMap) is set to value “PHYSICAL” from the vocabualry -->
+    <!-- CSIP82 mets/structMap/@LABEL The value must be “CSIP StructMap” -->
+    <!-- CSIP83 mets/structMap/@ID -->
+    <div ID="Structmap_Div_ID" LABEL="IP_18000_CSIP17_2">
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml]: Wrong LABEL value (maybe not the biggest mistake)

File: corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_3/METS.xml [hunk 6]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Minimal IP with schemas and no amdSec but a file in the folder metadata/preservation-->
+<!-- In this example IDs are carring information to ease understanding - they look like paths to ease understanding, but are just IDs, similar to the naming of namespaces - these IDs can be replaced with information less UUID -->
+<!-- CSIPSTR15 goes like this: We recommend including all schema documents for any structured metadata within package. These schema documents SHOULD be placed in a sub-folder called schemas within the Information Package root folder and/or the representation folder. -->
+<mets xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.loc.gov/METS/"
+  xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:csip="https://DILCIS.eu/XML/METS/CSIPExtensionMETS"
+  xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance schemas/XMLSchema.xsd http://www.loc.gov/METS/ schemas/mets.xsd 
+    http://www.w3.org/1999/xlink schemas/xlink.xsd 
+    https://DILCIS.eu/XML/METS/CSIPExtensionMETS schemas/CSIPExtensionMETS.xsd"
+    OBJID="IP_18000_CSIP17_3" TYPE="Database" csip:CONTENTINFORMATIONTYPE="SIARD2"
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_3/METS.xml]: Wrong OBJID value

File: corpora/csip/metadata/amdsec/CSIP31/testCase.xsd [hunk 7]
```diff
@@ -1 +1,185 @@
-../../../../../schema/testCase.xsd
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/testCase.xsd]: This is not the latest schema

File: corpora/csip/metadata/amdsec/CSIP31/test-case.xml [hunk 8]
```diff
@@ -1,24 +1,85 @@
 <!-- Root element for an individual test case, allows these to be wrapped into
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/test-case.xml]: Rename to testCase.xml

File: corpora/csip/metadata/amdsec/CSIP31/test-case.xml [hunk 9]
```diff
@@ -1,24 +1,85 @@
...
   <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP31"/>
   <!-- URL references to requirements for convenient lookup -->
   <references>
     <reference requirementId="CSIP31" URL="http://earkcsip.dilcis.eu/#CSIP31"/>
   </references>
   <!-- The full text of the requirement. -->
-  <requirementText>If administrative / preservation metadata is available, it must be described using the administrative metadata section (amdSec) element.
-It is possible to transfer metadata in a package using just the descriptive metadata sectiond/or adminstrative metadata section.
-</requirementText>
+  <requirementText><description>If administrative / preservation metadata is available, it must be described using the administrative metadata section (amdSec) element.
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP31/test-case.xml]: There is a wrong description-element

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
I have found some minor issues that need to be fixed, see comments.

File: corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml (hunk 4 above)
Comment (andersbonielsen) [reply corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml]: Fixed

File: corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml (hunk 5 above)
Comment (andersbonielsen) [reply corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml]: Fixed

File: corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_3/METS.xml (hunk 6 above)
Comment (andersbonielsen) [reply corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_3/METS.xml]: Fixed

File: corpora/csip/metadata/amdsec/CSIP31/test-case.xml (hunk 8 above)
Comment (andersbonielsen) [reply corpora/csip/metadata/amdsec/CSIP31/test-case.xml]: Fixed

File: corpora/csip/metadata/amdsec/CSIP31/testCase.xsd (hunk 7 above)
Comment (andersbonielsen) [reply corpora/csip/metadata/amdsec/CSIP31/testCase.xsd]: Backwards compatible, but now updated

File: corpora/csip/metadata/amdsec/CSIP31/test-case.xml (hunk 9 above)
Comment (andersbonielsen) [reply corpora/csip/metadata/amdsec/CSIP31/test-case.xml]: It is actually valid, looking at the new schema, I think we are supposed to use it like this. @koit

**Comment by carlwilson** [comment]
Fixed merge conflicts for final revew.

**Review by PhillipTommerholt — APPROVED** [review]

**Pull request merged**